docs/basic-features/data-fetching.md

```diff
@@ -660,7 +660,7 @@ export async function getServerSideProps(context) {
 The `context` parameter is an object containing the following keys:
 
 - `params`: If this page uses a dynamic route, `params` contains the route parameters. If the page name is `[id].js` , then `params` will look like `{ id: ... }`. To learn more, take a look at the [Dynamic Routing documentation](/docs/routing/dynamic-routes.md).
-- `req`: [The HTTP IncomingMessage object](https://nodejs.org/api/http.html#http_class_http_incomingmessage), with an additional `req.cookies` attribute – an object containing the cookies sent by the request (defaults to `{}`).
+- `req`: [The HTTP IncomingMessage object](https://nodejs.org/api/http.html#http_class_http_incomingmessage), plus [some built-in parsing](#built-in-contextreq-parsing).
```

Comment:
Lets try to be more explicit when we are describing things and linking to other places.

How about:
```suggestion
- `req`: [The HTTP IncomingMessage object](https://nodejs.org/api/http.html#http_class_http_incomingmessage), plus additional [built-in parsing helpers](#built-in-contextreq-parsing).
```